test/undo.t

```diff
@@ -68,6 +68,36 @@ class TestUndo(TestCase):
                                   expectedEnd=one_hour_before_utc,
                                   expectedAnnotation="")
 
+    def test_undo_annotate_with_embedded_quotes(self):
+        """Test undo of command 'annotate' with embedded quotes"""
+        now_utc = datetime.now().utcnow()
+        one_hour_before_utc = now_utc - timedelta(hours=1)
+        two_hours_before_utc = now_utc - timedelta(hours=2)
+
+        self.t("start {:%Y%m%dT%H%M%SZ} foo".format(two_hours_before_utc))
+        self.t("start {:%Y%m%dT%H%M%SZ} bar".format(one_hour_before_utc))
+        self.t("annotate 'foo \"bar\"'")
+
+        j = self.t.export()
+        self.assertEqual(len(j), 2, msg="Expected 2 intervals before, got {}".format(len(j)))
+        self.assertClosedInterval(j[0],
+                                  expectedStart=two_hours_before_utc,
+                                  expectedEnd=one_hour_before_utc,
+                                  expectedTags=["foo"])
+        self.assertOpenInterval(j[1],
+                                expectedStart=one_hour_before_utc,
+                                expectedTags=["bar"],
+                                expectedAnnotation="foo \"bar\"")
+
+        self.t("undo")
+
+        j = self.t.export()
+        self.assertOpenInterval(j[1],
```

Comment:
In all other tests you test whether the interval count is correct (e.g. line 477), here it is missing... 😉